pkg/handler/v2/routes_v2.go

```diff
@@ -64,6 +64,20 @@ func (r Routing) RegisterV2(mux *mux.Router, metrics common.ServerMetrics) {
 		Path("/featuregates").
 		Handler(r.getFeatureGates())
 
+	// Defines a set of HTTP endpoints for interacting with EKS clusters
+	mux.Methods(http.MethodGet).
+		Path("/providers/eks/clusters").
+		Handler(r.listEKSClusters())
+
+	mux.Methods(http.MethodGet).
+		Path("/providers/ec2/regions").
+		Handler(r.listEC2Regions())
```

Comment:
please move this endpoint to `	
// Defines a set of HTTP endpoint for interacting with
// various cloud providers`
 and change for `/providers/aws/regions`

Reply:
done